### PR TITLE
cmd: add log to http proxy environment variables

### PR DESCRIPTION
**Pull request by YuJuncen**
### What problem does this PR solve? 
slove #1482

### What is changed and how it works?
Since gRPC uses `https_proxy` already(see [here](https://github.com/grpc/grpc-go/blob/master/Documentation/proxy.md), even with a [caveat](https://github.com/pingcap/ticdc/issues/1482#issuecomment-794988747)), we only need to hint the user that those variables was used.

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)

#### cli
image

#### server
image

### Release note

- cmd: commands would environment variables relative to proxies now.

### CODE REVIEW

**Comment by YuJuncen** [comment]
PD disconnected...?

/run_integration_test

File: cmd/util.go [hunk 1]
```diff
@@ -311,3 +313,21 @@ func strictDecodeFile(path, component string, cfg interface{}) error {
 
 	return errors.Trace(err)
 }
+
+// logHTTPProxies logs HTTP proxy relative environment variables.
+func logHTTPProxies() {
```
Comment (amyangfei) [inline cmd/util.go:318]: Please add a unit test for this function

Comment (YuJuncen) [reply cmd/util.go:318]: Done.

**Comment by YuJuncen** [comment]
Can I run it?

/run-integration-tests

**Comment by YuJuncen** [comment]
Finally... PTAL @leoppro @amyangfei

**Comment by YuJuncen** [comment]
/run-all-tests

It fails consistently? 🧐

**Pull request merged**